apps/dataflow/src/scraper/fms/get_all_svg.py

```
@@ -0,0 +1,241 @@
+
+3. Update the `cookies` and `headers` dicts in this script with the values copied from your browser session.
+
+4. Run this script to fetch all SVGs
+"""
+
+import argparse
+import json
+from pathlib import Path
+import re
+import time
+from typing import Dict, Any, List, Optional
```

Ruff (0.14.14)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)